gui/src/containers/Bridges/Edit.js

```diff
@@ -96,8 +93,8 @@ export class Edit extends Component {
                 url={bridge.url}
                 confirmations={bridge.confirmations}
                 minimumContractPayment={bridge.minimumContractPayment}
-                onSuccess={successNotification}
-                onError={errorNotification}
+                onSuccess={SuccessNotification}
+                onError={ErrorMessage}
```

Comment:
It took me a while to mentally pinpoint what was throwing me off with these names. Through habit, I've followed the convention that functions are named as verbs. For example, below we have `mapStateToProps` which is clearly a function. `SuccessNotification` and `ErrorMessage`, however, sound like classes or instances, but are functions. To follow the functions are verbs convention, `NotifySuccess` and `CreateErrorMessage` would be more apt and is what I would recommend.


The discontinuity is better illustrated in the next use of `Form`:

```jsx
            onSubmit={props.createJobSpec}
            onSuccess={SuccessNotification}
            onError={ErrorMessage}
```

We're going from `createJobSpec` to `SuccessNotification`. `c(C)reateSuccessNotification` or `NotifySuccess` is better imo.

Reply:
Then there's the convention of having the same name as the file, which is an issue with error:

`import ErrorMessage from 'components/Notifications/DefaultError'`

which is probably what swayed you to this direction. Is there a way reconcile the two?

Reply:
Gah! `ErrorMessage` should be -> `DefaultError`